README.md

```diff
@@ -1,58 +1,117 @@
 
-Windows Platform tools are required for building native binaries.
-You can install them in a shell opened as administrator with the following snippet:
+Windows Platform tools are required for building native binaries. You can install them in a shell opened as administrator with the following snippet:
 
-```bash
+```
 npm install --global --production windows-build-tools
 ```
 
 **Note: This will take some time and you don't get much feedback of what is happening, so please wait until you get confirmation the installation process has finished otherwise painful Visual Studio things will need to be fixed!**
 
 #### 'vue-cli-service' is not recognized
 
-Please see issue 122 (https://github.com/vcync/modv-3/issues/122#issuecomment-640100114) on how to resolve this.
-
+Please see issue 122 ([vcync/modv-3/issues/122#issuecomment-640100114](https://github.com/vcync/modv-3/issues/122#issuecomment-640100114)) on how to resolve this.
 
 ### Ubuntu/Debian
 
-libndi is required for NDI sources and must be installed for modV to build. You can find that available to download here:
-[https://github.com/Palakis/obs-ndi/releases](https://github.com/Palakis/obs-ndi/releases)
+libndi is required for NDI sources and must be installed for modV to build. You can find that available to download here: [Palakis/obs-ndi/releases](https://github.com/Palakis/obs-ndi/releases)
 
 Last successful build was with `libndi4_4.5.1-1_amd64.deb`.
 
+### Other Linux flavours
+
+Untested. NDI is provided by grandiose, our fork is here: [vcync/grandiose](https://github.com/vcync/grandiose/) This fork of grandiose has other libndi supported platforms, however even on Ubuntu we needed the above libndi package to be installed.
+
+Let us know how you get on (good or bad) and we'll update the repo and docs accordingly.
+
+
+
+## Contributing
+
+Contributions, issues and feature requests are welcome!
+Feel free to check [issues page](https://github.com/vcync/modV/issues).
+
+
+
+## Show your support
+
+Give a ⭐️ if this project helped you!
+
+
+
+## Acknowledgements
+
+Thank you to:
+
+- [Tim Pietrusky](https://nerddis.co/) for his continued support, help, mentorship and kindness
+- [Live:JS](http://livejs.network/) for inspiration, motivation and advice
+- Hugh Rawlinson, Nevo Segal and Jakub Fiala for the incredible audio analysis engine, [meyda](https://github.com/hughrawlinson/meyda)
+- [Dario Villanueva](http://alolo.co/) for his advice and introduction to live visuals which inspired this whole project
 
 
-### Other Linux flavours
 
-Untested. NDI is provided by grandiose, our fork is here: [https://github.com/vcync/grandiose/](https://github.com/vcync/grandiose/)
-This fork of grandiose has other libndi supported platforms, however even on Ubuntu we needed the above libndi package to be installed.
+## License
 
-Let us know how you get on (good or bad) and we'll update the repo and docs accordingly.
\ No newline at end of file
+Copyright © 2020 [Sam Wray](https://github.com/vcync).
```

Comment:
Why not as `vcync`?

Reply:
I used a generator for the basic template. Didn't realise it had changed this - I'll update!